wire-ios-data-model/Source/MLS/MLSService.swift

```diff
@@ -544,7 +548,8 @@ public final class MLSService: MLSServiceInterface {
         do {
             logger.info("adding members to group (\(groupID.safeForLoggingDescription)) with users: \(users)")
             guard !users.isEmpty else { throw MLSAddMembersError.noMembersToAdd }
-            let keyPackages = try await claimKeyPackages(for: users)
+            guard let ciphersuite = MLSCipherSuite(rawValue: await featureRepository.fetchMLS().config.defaultCipherSuite.rawValue) else { throw MLSAddMembersError.noMembersToAdd}
```

Comment:
```suggestion
            let mlsConfig = await featureRepository.fetchMLS().config
            guard let ciphersuite = MLSCipherSuite(rawValue: mlsConfig.defaultCipherSuite.rawValue) else {
                throw MLSAddMembersError.noMembersToAdd
            }
```

it might be easier to read on multiple lines